File: programme/volume-of-cube/volume-of-cube.c
// Write a programme to find volume of a cube
#include <stdio.h>

int Vol_cube(int s)
{
  int Volume = s * s * s;
  return Volume;
}

int main()
{
  int side;
  printf("Enter cube side : ");
  scanf("%d", &side);

  printf("\nVolume of cube : %d\n", Vol_cube(side));

  return 0;
}

File: programme/volume-of-cube/volume-of-cube.cs
// Write a programme to find volume of a cube
using System;

namespace Codinasion.Programme
{
    public class VolumeOfCube
    {
        public static void Main()
        {
            Console.Write("Enter Side : ");
            decimal Side = Convert.ToDecimal(Console.ReadLine());
            decimal Volume = Side * Side * Side;
            Console.WriteLine("\nVolume of cube : " + Volume);
        }
    }
}
